tests/reporting/test_market_unit.py

```diff
@@ -0,0 +1,527 @@
+
+from ethereum.tools import tester
+from datetime import timedelta
+from utils import longToHexString, stringToBytes, bytesToHexString
+from pytest import fixture, raises
+from ethereum.tools.tester import ABIContract, TransactionFailed
+
+
+def test_market_creation(localFixture, mockUniverse, mockReportingWindow, mockCash, chain, constants, mockMarket, mockReputationToken, mockShareToken, mockShareTokenFactory):
+    numTicks = 10 ** 10
+    fee = 1
+    oneEther = 10 ** 18
+    endTime = chain.head_state.timestamp + constants.DESIGNATED_REPORTING_DURATION_SECONDS()
+    market = localFixture.upload('../source/contracts/reporting/Market.sol', 'market')
+    market.setController(localFixture.contracts["Controller"].address)
+    with raises(TransactionFailed, message="outcomes has to be greater than 1"):
+        market.initialize(mockReportingWindow.address, endTime, 1, numTicks, fee, mockCash.address, tester.a1, tester.a1)
+
+    with raises(TransactionFailed, message="outcomes has to be less than 9"):
+        market.initialize(mockReportingWindow.address, endTime, 9, numTicks, fee, mockCash.address, tester.a1, tester.a1)
+
+    with raises(TransactionFailed, message="numTicks needs to be divisable by outcomes"):
+        market.initialize(mockReportingWindow.address, endTime, 7, numTicks, fee, mockCash.address, tester.a1, tester.a1)
+
+    with raises(TransactionFailed, message="fee per eth can not be greater than max fee per eth in attoEth"):
+        market.initialize(mockReportingWindow.address, endTime, 5, numTicks, 5, mockCash.address, tester.a1, tester.a1)
```

Comment:
I don't think this is failing because of the fee max requirement. Note the max is denominated in `ether` so it's actually 10**18 / 2. Also the test below are using this fee amount, which while fine (I think) would be wrong for the test if this was actually failing because of the fee amount

Reply:
I'll make the correction to passed in onEther / 2 + 1.